src/viam/ur/module/ur_arm_state_controlled.cpp

```diff
@@ -62,14 +62,16 @@ std::optional<URArm::state_::event_variant_> URArm::state_::state_controlled_::h
         return std::nullopt;
     }
 
-    if (!state.move_request_->samples.empty() && !state.move_request_->cancellation_request) {
-        // We have a move request, it has samples, and there is no pending cancel for that move. Issue the move.
+    if ((state.move_request_->trajectory_start.time_since_epoch().count() == 0) && !state.move_request_->cancellation_request) {
```

Comment:
switched to using time here instead of emptying the samples, so that when a failure occurs I can pull the samples from the move request.